examples/tensorflow-resnet/README.md

```diff
@@ -0,0 +1,187 @@
+```bash
+tele build -o tensorflowresnet.tar tensorflow-resnet/resources/charts/tensorflow-resnet
+```
+
+## Deployment Configuration
+The Tensorflow-ResNet application runs in the default configuration with two ports, 8500, 8501.  The 8500 port is used for client image classification requests and configured available on a NodePort of 30090 by default.  
+
+After deploying you will see the following Pod status of initalizing while the model is loaded.
+
+```bash
+$ kubectl get po
+NAME                                READY   STATUS            RESTARTS   AGE                                                                                                                                                              
+tensorflow-resnet-bf8f6f5b6-22pgd   0/1     PodInitializing   0          7s                                                                                                                                                               
+tensorflow-resnet-bf8f6f5b6-59fq8   0/1     PodInitializing   0          7s                                                                                                                                                               
+tensorflow-resnet-bf8f6f5b6-qbd6v   0/1     PodInitializing   0          7s    
+```
+
+Once the model is loaded these pods are available to serve image classifications.
+```bash
+kubectl get po                                                                                                                                                                                         
+NAME                                READY   STATUS    RESTARTS   AGE                                                                                                                                                                      
+tensorflow-resnet-bf8f6f5b6-22pgd   1/1     Running   0          37s                                                                                                                                                                      
+tensorflow-resnet-bf8f6f5b6-59fq8   1/1     Running   0          37s                                                                                                                                                                      
+tensorflow-resnet-bf8f6f5b6-qbd6v   1/1     Running   0          37s
+```
+```bash
+$ kubectl get deployments
+NAME                READY   UP-TO-DATE   AVAILABLE   AGE
+tensorflow-resnet   3/3     3            3           10m
+```
+
+# Image classification
+
+Image classification requests should be done on a machine with Docker installed. Take note of one of the server's available IPs and confirm the available NodePort.  By default it should be 30090. After invoking you will recieve a result tensor.  
```

Comment:
~~After invoking you will receive a result tensor~~ I think this does not provide additional information beyond what you've already given.